package.json

```diff
@@ -15,6 +15,7 @@
     "@astrojs/netlify": "^2.2.3",
     "@astrojs/node": "^5.3.3",
     "@astrojs/react": "^2.2.0",
+    "@lexical/react": "^0.12.2",
```

Comment:
I guess the Lexical dependencies can be removed, right?

Reply:
Done